src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/model/common/Event.java

```diff
@@ -6,6 +6,7 @@
     ZIPFILE_PROCESSING_STARTED, // when processor starts processing zipfile from blob
     DOC_FAILURE, // generic failure while processing zip file. before uploading to document management
     FILE_VALIDATION_FAILURE,
+    SERVICE_DISABLED,
```

Comment:
SERVICE_DISABLED is name for a property. Name for an event should be DISABLED_SERVICE_FAILURE or something like this.

Reply:
Renamed